<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelecJugadores</class>
 <widget class="QDialog" name="SelecJugadores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Jugador 4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ltx_jugador4">
      <property name="tabletTracking">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Icono</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbx_jugador4"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Jugador 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ltx_jugador2"/>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Icono</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbx_jugador2"/>
    </item>
   </layout>
   <zorder>label_2</zorder>
   <zorder>label_7</zorder>
   <zorder>cbx_jugador2</zorder>
   <zorder>ltx_jugador2</zorder>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>59</y>
     <width>511</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Jugador 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ltx_jugador1"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Icono</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbx_jugador1"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Jugador 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ltx_jugador3">
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Icono</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbx_jugador3"/>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="btnBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>280</y>
     <width>167</width>
     <height>29</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Personalizar jugadores</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
